cachito/web/validation.py

```diff
@@ -46,3 +50,47 @@ def validate_dependency_replacements(replacements: List[Dict[str, Any]]) -> None
         raise ValidationError('"dependency_replacements" must be an array')
     for replacement in replacements:
         validate_replacement(replacement)
+
+
+class RequestBodyValidator(decorators.validation.RequestBodyValidator):
```

Comment:
These classes do not have type annotations... although the base classes also don't, so it's probably fine.